apps/pages/templates/pages/review_guide.html

```diff
@@ -139,9 +139,9 @@ <h2 id="review-faq">{{ _('Frequently Asked Questions about Reviews') }}</h2>
 
     <h3>{{ _('I’m an add-on author, can I respond to reviews?') }}</h3>
     <p>
-      {% trans %}
+      {% trans forums_url='https://discourse.mozilla-community.org/c/add-ons/add-on-support' %}
         Yes, add-on authors can provide a single response to a review. You can 
-        set up a discussion topic in our <a href="http://forums.mozilla.org/viewforum.php?f=23">forum</a> 
+        set up a discussion topic in our <a href="https://discourse.mozilla-community.org/c/add-ons/add-on-support">forum</a> 
```

Comment:
This would need to rename all the `%(forums_url)s` in the `.po` files also (if we do want to update those manually).